core/dbt/task/run.py

```diff
@@ -170,9 +170,8 @@ def get_node_representation(self):
         return str(relation)
 
     def describe_node(self):
-        return "{} model {}".format(
-            self.node.get_materialization(), self.get_node_representation()
-        )
+        # TODO CL 'language' will be moved to node level when we change representation
```

Comment:
👍

Reply:
Yes. Config is usually things the user can set, and I think 'language' is not.

Reply:
Yea, I have a PR with this coming, the internal representation one, moving it to node level. Currently that PR is on hold for team dependency reason